lib/stylelint/processorRemoveMixins.js

```diff
@@ -0,0 +1,7 @@
+// If this proposed functionality ever gets added https://github.com/styled-components/stylelint-processor-styled-components/issues/271
```

Comment:
Does https://github.com/styled-components/stylelint-processor-styled-components/issues/271#issuecomment-511561073 not work?

Reply:
Unfortunately it doesn't tell the CSS validator plugin to ignore those custom properties and rules. That's why the plugin has it's own ignore option which doesn't support regular expressions like stylelint's rules do. I have [suggested this functionality in csstree/stylelint-validator](https://github.com/csstree/stylelint-validator/issues/19)